docker/run-apache2.sh

```diff
@@ -3,6 +3,8 @@
 _DECK_HOST=$DECK_HOST
 _DECK_PORT=$DECK_PORT
 _API_HOST=$API_HOST
+_DECK_CERT=$DECK_CERT
```

Comment:
Yes, they're paths. From what I understand, these env vars (https://github.com/spinnaker/deck/blob/master/webpack.common.js#L101) are supposed to be paths, because `readFileSync` takes a path as the argument (https://nodejs.org/api/fs.html#fs_fs_readfilesync_file_options). Is this understanding correct?

So, I've renamed the local variables but not the env variables, because I don't want to break existing usages.